packages/riipen-ui-docs/src/pages/components/table/Mobile.jsx

```diff
@@ -0,0 +1,53 @@
+import React from "react";
+
+import Table from "@riipen-ui/components/Table";
+import TableColumn from "@riipen-ui/components/TableColumn";
+
+export default function Mobile() {
```

Comment:
Is there anyway we can simplify this documentation? I've read it 3 times and still feel like I don't totally know what's going on. Documentation should be verbose over clever to show what's really going on IMO.